encrypted-media/scripts/playback-temporary-waitingforkey.js

```diff
@@ -72,19 +54,18 @@ function runTest(config) {
             return access.createMediaKeys();
         }).then(function(mediaKeys) {
             _mediaKeys = mediaKeys;
-
             _video.setMediaKeys(_mediaKeys);
-
             waitForEventAndRunStep('waitingforkey', _video, onWaitingForKey, test);
-
+            
+            // Not using waitForEventAndRunStep() to avoid too many
```

Comment:
I'll add an assert in onTimeupdate to check we have the expected number of sessions.